### ISSUE TITLE
Raven reader only displays two of my articles 

### ISSUE THREAD

**Issue by S3j5b0** [hello-efficiency-inc/raven-reader#546]
**Describe the bug**
I am trying to localhost my own little application that exposes a very simple rss feed xml file. I then want ravenreader to read this feed.
However ravenreader seems to consistently ignore some of my articles. It works fine on other rss reader services.

**To Reproduce**
I create an application that exposes the following feed

```
<rss version="2.0">
<channel>
<title>Andrew's PyRSS2Gen feed</title>
<link>http://www.dalkescientific.com/Python/PyRSS2Gen.html</link>
<description>The latest news about PyRSS2Gen, a Python library for generating RSS2 feeds</description>
<lastBuildDate>Fri, 09 Apr 2021 13:09:46 GMT</lastBuildDate>
<generator>PyRSS2Gen-1.1.0</generator>
<docs>http://blogs.law.harvard.edu/tech/rss</docs>
<item>
<title>PyRSS2Gen-0.0 released</title>
<link>http://www.dalkescientific.com/news/030906-PyRSS2Gen.html</link>
<description>Ddddddwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww </description>
<guid isPermaLink="true">http://www.dalkescientific.com/news/030906-PyRSS2Gen.html</guid>
<pubDate>Sat, 06 Sep 2003 21:31:00 GMT</pubDate>
</item>
<item>
<title>Thoughts on RSS feeds for bioinformatics</title>
<link>http://www.dalkescientific.com/writings/diary/archive/2003/09/06/RSS.html</link>
<description>lorem ipsum suy ceck </description>
<guid isPermaLink="true">http://www.dalkescientific.com/writings/diary/archive/2003/09/06/RSS.html</guid>
<pubDate>Sat, 06 Sep 2003 21:49:00 GMT</pubDate>
</item>
</channel>
</rss>
```

I then open ravenreader, and tells it to subscribe to http://0.0.0.0:8000/ where I have exposed the file.

But now  when I create the subscription, ravenreader only displays the two following articles:

![image](https://user-images.githubusercontent.com/44634922/114198111-3fffdc00-9953-11eb-83b6-f9213e59a18d.png)

Why would this file be displayed in this wrong manner?

**Desktop (please complete the following information):**
 - Ubuntu 20
 - node 14.16
 - and localhosting the application with a forked repo

**Comment by mrgodhani** [comment]
@S3j5b0 Do you have OPML file so I can test?  This is all dependent on https://github.com/rbren/rss-parser. This is the package we use.

**Comment by S3j5b0** [comment]
yes i looks the following way, git doesnt like me uploading xml files for some reason:

<?xml version="1.0" encoding="iso-8859-1"?>
<rss version="2.0"><channel><title>Andrew's PyRSS2Gen feed</title><link>http://www.dalkescientific.com/Python/PyRSS2Gen.html</link><description>The latest news about PyRSS2Gen, a Python library for generating RSS2 feeds</description><lastBuildDate>Fri, 09 Apr 2021 16:19:00 GMT</lastBuildDate><generator>PyRSS2Gen-1.1.0</generator><docs>http://blogs.law.harvard.edu/tech/rss</docs><item><title>hardcoded fedeed</title><link>http://www.dalkescientific.com/news/030906-PyRSS2Gen.html</link><description>Ddddddwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww </description><guid isPermaLink="true">http://www.dalkescientific.com/news/030906-PyRSS2Gen.html</guid><pubDate>Sat, 06 Sep 2003 21:31:00 GMT</pubDate></item><item><title>Thoughts on RSS feeds for bioinformatics</title><link>http://www.dalkescientific.com/writings/diary/archive/2003/09/06/RSS.html</link><description>lorem ipsum suck my ceck </description><guid isPermaLink="true">http://www.dalkescientific.com/writings/diary/archive/2003/09/06/RSS.html</guid><pubDate>Sat, 06 Sep 2003 21:49:00 GMT</pubDate></item><item><title> HEI ddddddd test test testTLER</title><link>http://www.dalkescientific.com/writings/diary/archive/2003/09/06/RSS.html</link><description>lidngle  </description><guid isPermaLink="true">https://www.youtube.com/watch?v=Jr-PuuKYn8c</guid><pubDate>Wed, 06 Aug 2003 21:49:00 GMT</pubDate></item></channel></rss>
